_docs/aws_resources.md

```diff
@@ -58,7 +58,38 @@ How to create EC2 instance in AWS step by step, including creating the EC2 resou
 [EC2 Video Tutorial Available Here](https://www.youtube.com/watch?v=1x5dWRaLvIw)
 
 ## AWS EC2 Linux Launch & Connection Using SSH
-
 This video will show how to use a PuTTY private key to connect to your Amazon EC2 Linux instance.
 
-* [EC2 Linux Launch Video Available Here](https://www.youtube.com/watch?v=v0g1M5bb9u4)
+[EC2 Linux Launch Video Available Here](https://www.youtube.com/watch?v=v0g1M5bb9u4)
+
+
+
+# AWS IDEs/ Development tools
+
+**AWS Cloud9**
+
+AWS Cloud9 is a cloud-based integrated development environment (IDE) that lets you write, run, and debug your code with just a browser. It includes a code editor, debugger, and terminal. It can support multiple languages including but not limited to Java, C++, Python, Javascript, etc. One big advantage of using Cloud9 is that you can quickly set it up and start coding right away without the need to install addtional materials. However, you can still choose to use your own IDE if you prefer. AWS also supports other IDEs like Eclipse, PyCharm, Visual Studio Code.
```

Comment:
```suggestion
AWS Cloud9 is a cloud-based integrated development environment (IDE) that lets you write, run, and debug your code with just a browser. It includes a code editor, debugger, and terminal. It can support multiple languages including but not limited to Java, C++, Python, Javascript, etc. One big advantage of using Cloud9 is that you can quickly set it up and start coding right away without the need to install addtional materials. However, you can still choose to use your own IDE if you prefer. AWS also supports other IDEs like Eclipse, PyCharm, Visual Studio Code.
```